watchdog/src/remote_api/blockstream_info.rs

```diff
@@ -0,0 +1,68 @@
+use crate::remote_api::http::{create_request, fetch_body};
+use crate::remote_api::storage;
+use crate::types::BlockHeight;
+use ic_cdk::api::management_canister::http_request::CanisterHttpRequestArgument;
+
+pub struct BlockstreamInfo {}
```

Comment:
nit: do we need this struct? Is there an advantage to having this struct vs just having the methods below as pure functions?